crates/ruma-client-api/src/r0/knock/knock_room.rs

```diff
@@ -0,0 +1,50 @@
+//! [POST /_matrix/client/r0/knock/{roomIdOrAlias}](https://github.com/Sorunome/matrix-doc/blob/soru/knock/proposals/2403-knock.md)
```

Comment:
Since this is documented in the unstable spec, please link to that: https://spec.matrix.org/unstable/client-server-api/#post_matrixclientr0knockroomidoralias